project/REVIEWING.md

```diff
@@ -0,0 +1,170 @@
+Pull request reviewing process
+==============================
+
+# Workflow
+
+An opened pull request can be in 1 of 5 distinct states, for each of which there is a corresponding
+label that needs to be applied:
+
+## Triage - `status/0-triage`
```

Comment:
its too bad we can't have these start with a letter that would move them to the top of the drop-down list  :-)